frontend/app/route-helpers/apply-route-helpers.server.ts

```diff
@@ -224,13 +226,6 @@ function validateStateForReview({ params, state }: ValidateStateForReviewArgs) {
     throw redirect(getPathById('$lang+/_public+/apply+/$id+/date-of-birth', params));
   }
 
-  const parseDateOfBirth = parse(state.dateOfBirth, 'yyyy-MM-dd', new Date());
-  const age = differenceInYears(new Date(), parseDateOfBirth);
-
-  if (age < 65) {
```

Comment:
We should still validate if you're under 18 years old, right?

Reply:
based on Figma, there a few scenarios:  `age>=65`, `18<=age<=64`, `16<=age<=17`, `age<16`.  So I forsee there needing to be changes to `route-helpers.server.ts` in future PRs.

Reply:
Make sure which scenarios are included for June release.

Reply:
all of the ones I listed above.  Each screen is it's own ADO card.

![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/48450599/013a83d6-9876-449d-94ad-cb2674d864a8)

Reply:
No fix because the under 18 scenario will be integrated for June release.